ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/dialogs/RestorePurchasesDialog.kt

```diff
@@ -26,14 +26,16 @@ import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.CustomerCent
 internal fun RestorePurchasesDialog(
     state: RestorePurchasesState,
     localization: CustomerCenterConfigData.Localization,
-    onDismiss: () -> Unit,
+    onDismiss: (restored: Boolean) -> Unit,
     onRestore: () -> Unit,
     onContactSupport: (() -> Unit)?,
 ) {
     when (state) {
         RestorePurchasesState.INITIAL -> InitialStateDialog(
             localization = localization,
-            onDismiss = onDismiss,
+            onDismiss = {
+                onDismiss(false)
+            },
             onRestore = onRestore,
         )
         RestorePurchasesState.PURCHASES_RECOVERED -> PurchasesRecoveredDialog(
```

Comment:
Not from this PR but I was looking at the logic behind knowing whether purchases were recovered. It seems we just check if there are any active subs or any non subs at all... I do wonder if we can consider that purchases were recovered in that case, since those might be purchases from other stores, and nothing was recovered from the current store for example... 

I guess to get this properly, we would need some sort of diff between the customer info before/after the restore, but that's a bit trickier... Nothing to do in this PR, but something to consider about the behavior here.